### PR TITLE
WIP radiasfot/raydata#27: dispaly completed scans

### PR DESCRIPTION
**Pull request by e-carlin**
depends on radiasoft/raydata#31

### CODE REVIEW

File: sirepo/package_data/static/js/raydata.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ SIREPO.app.config(() => {
...
         </div>
-        <div data-ng-switch-when="QueuedScansTable" class="col-sm-12">
-          <div data-scans-table="" data-model-name="modelName"></div>
-        </div>
         <div data-ng-switch-when="CompletedScansTable" class="col-sm-12">
-          <div data-scans-table-with-modal="" data-model-name="modelName"></div>
+          <div data-scans-table="" data-model-name="modelName" data-analysis-status="executed"></div>
+        </div>
+        <div data-ng-switch-when="QueuedScansTable" class="col-sm-12">
+          <div data-scans-table="" data-model-name="modelName" data-analysis-status="queued"></div>
```
Comment (rorour) [inline sirepo/package_data/static/js/raydata.js:15]: The modal is on the queued tab as well, is that what we want?

Comment (e-carlin) [reply sirepo/package_data/static/js/raydata.js:15]: Yes for running scans it is nice to see the output they have produced so far.

File: sirepo/template/raydata.py [hunk 2]
```diff
@@ -93,12 +76,15 @@ def stateless_compute_catalog_names(data):
 def stateless_compute_scans(data):
     # TODO(e-carlin): get scans from daemon
     l = []
-    if data.scansStatus == "completed_scans":
+    if data.analysisStatus == "executed":
         assert data.searchStartTime and data.searchStopTime, pkdformat(
             "must have both searchStartTime and searchStopTime data={}", data
         )
-        l = [ScanObject("uid1"), ScanObject("uid2"), ScanObject("uid3")]
```
[the remark this answers is not in the document]
Comment (e-carlin) [reply sirepo/template/raydata.py:100]: 1c8e660fc

File: sirepo/template/raydata.py [hunk 3]
```diff
@@ -93,12 +76,15 @@ def stateless_compute_catalog_names(data):
 def stateless_compute_scans(data):
     # TODO(e-carlin): get scans from daemon
     l = []
```
[the remark this answers is not in the document]
Comment (e-carlin) [reply sirepo/template/raydata.py]: 6ae20b236

**Comment by e-carlin** [comment]
@rorour I'm going to fix some naming discrepancies and one TODO. I'll re-request a review when done.

**Comment by e-carlin** [comment]
Ready for another review.

**Review by rorour — APPROVED** [review]
Looks good

**Pull request merged**